components/automate-ui/src/app/pipes/pagination-true-end.pipe.ts

```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'paginationTrueEnd'
+})
+export class PaginationTrueEndPipe implements PipeTransform {
+
+  transform(pageEnd: number, totalCount: number): number {
+    if (pageEnd > totalCount) {
+      pageEnd = totalCount;
+    }
+    return pageEnd;
```

Comment:
More functional (and more concise):
```suggestion
    return (pageEnd > totalCount) ? totalCount : pageEnd;
```

Reply:
nice refactor!